src/Webservice/Webservice.php

```diff
@@ -168,7 +168,7 @@ public function describe($endpoint)
         $shortName = App::shortName(get_class($this), 'Webservice', 'Webservice');
         list($plugin, $name) = pluginSplit($shortName);
 
-        $schemaShortName = implode('.', array_filter([$plugin, Inflector::classify($endpoint)]));
+        $schemaShortName = implode('.', array_filter([$plugin, Inflector::classify(Inflector::underscore($endpoint))]));
```

Comment:
Using str_replace() to change the dash to underscore would be more efficient since we know rest of the string is already going to be lowercased.